mllam_data_prep/config.py

```diff
@@ -52,6 +52,24 @@ class ValueSelection:
     units: str = None
 
 
+@dataclass
+class DerivedVariable:
+    """
+    Defines a derived variables, where the kwargs (variables required
+    for the calculation) and the function (for calculating the variable)
+    are specified.
+
+    Attributes:
+        kwargs: Variables required for calculating the derived variable.
+        function: Function used to calculate the derived variable.
+        attributes: Attributes (e.g. `units` and `long_name`) for the derived variable.
```

Comment:
I don't have any strong feelings if the call it `attributes` or `attrs` but I think I prefer `attrs` (since it is shorter and still easily understood what it means). I have changed it now :)